draft-connolly-cfrg-xwing-kem.md

```diff
@@ -117,50 +119,191 @@ informative:
 
 --- abstract
 
-This document specifies a general-purpose key encapsulation mechanism (KEM) that
-commits to public key material and ciphertext material as part of computing the
-shared secret. KEMs with these properties are attractive for protocols that are
-looking to replace pure Diffie-Hellman key agreement.
-
+This memo defines X-Wing, a general-purpose post-quantum/traditional
+    hybrid key encapsulation mechanism (PQ/T KEM)
+    built on X25519 and ML-KEM-768.
 
 --- middle
 
 # Introduction
 
-A KEM is a three-tuple of algorithms (*KeyGen*, *Encapsulate*, *Decapsulate*):
+## Warning: ML-KEM-768 has not been standardised
+
+X-Wing uses ML-KEM-768, which has not been standardised yet.
+Thus X-Wing is not finished, yet, and should not be used, yet.
+
+## Motivation
+
+There are many choices that can be made when
+    specifying a hybrid KEM:
+    the constituent KEMs;
+    their security levels;
+    the combinber;
+    and the hash within, to name but a few.
+Having too many similar options are a burden to the ecosystem.
+The aim of X-Wing is to provide a concrete, simple choice
+    for post-quantum hybrid KEM,
+    that should be suitable for the vast majority of use cases.
+
+## Design goals
+
+1. By making concrete choices, we can simplify and improve  many
+   aspects of X-Wing as compared to a more generic combiner.
+
+    - Simplicity of definition. For one, because all shared secrets
+       and cipher texts are fixed length, we do not need to encode the length.
+
+    - Its security analysis. Because ML-KEM-768 already assumes QROM, we
+       do not need to complicate the analysis of X-Wing by considering
+       weaker models.
+
+    - Its performance. By using SHA3-256 in the combiner, which matches
```

Comment:
sha3 also is more secure than sha2 and avoids requiring hkdf

Reply:
Technically SHA3 is more secure, but it feels to me like stating that a 40,000kg rock is harder to lift by a person than a 10,000kg rock. Both are impossible.
The hkdf point is a good one.